static.config.js

```diff
@@ -3,10 +3,10 @@ import { request } from 'graphql-request';
 
 // QUERIES
 import allServicePagesQuery from 'js/queries/allServicePagesQuery';
-import servicePageQuery from 'js/queries/servicePageQuery';
 import allTopicPagesQuery from 'js/queries/allTopicPagesQuery';
 import topicPageQuery from 'js/queries/topicPageQuery';
```

Comment:
Minor comments for now as I wrap my head around this stuff.
Can we get rid of these single page queries (Topic, Service)?
Can we rename departmentPageQuery to AllDepartmentPagesQuery to match the others?
Do we use js/helpers/fetchData.js any longer too?

Reply:
yes, yes, nope